doc/examples.rst

```diff
@@ -152,3 +152,123 @@ Instantiated ``init.lua`` content:
 
     require("fiber").sleep(1)
 
+Packing environments
+----------------------------------
+
+For example, we want to pack a single application. If there is the next environment structure::
+      single_environment/
+      ├── tarantool.yaml
+      └── init.lua
+
+The contents of ``./tarantool.yaml`` is supposed to be:
```

Comment:
"The content"
Or just use a file name:
``tarantool.yaml``:
Also, I think it is better to omit ./ before file name.

Reply:
Fixed.